cfme/common/provider_views.py

```diff
@@ -260,10 +260,13 @@ class ProviderTemplatesView(ProviderVmsTemplatesView):
 
     @property
     def is_displayed(self):
-        title = VersionPicker({
-            Version.lowest(): '{name} (All VM Templates)'.format(name=self.context['object'].name),
-            '5.10': '{name} (All VM Templates and Images)'.format(name=self.context['object'].name),
-        }).pick(self.extra.appliance.version)
+        expected_name = getattr(
+            self.context['object'], 'name', None) or getattr(
+            self.context['object'].parent, 'name', None)
+        title = VersionPicker(
+            {Version.lowest(): f'{expected_name} (All VM Templates)',
+            '5.10': f'{expected_name} (All VM Templates and Images)'
```

Comment:
Didn't catch this earlier, but this no longer needs to be version picked. Lowest supported version of CFME in integration_tests is 5.10. Its okay to leave it in, but note its not necessary to include the 5.9 string here via Version.lowest.